### PR TITLE
Analytics tracking for the employers page

### LINKED ISSUE #453: Mixpanel funnel for Employers

### PR DESCRIPTION
**Pull request by pitchinvasion**
This is the code I started working on before I left for South Africa but I'm now ready to submit it as I've completed the funnel with the code on the applications repo that tracks when they actually submit a typeform.

Closes #453

### CODE REVIEW

File: source/javascripts/link_tracking.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
+ * Tracks clicks to external links (for the moment just on the employers page)
+ */
+(function ($) {
```
Comment (silvabox) [inline source/javascripts/link_tracking.js]: @pitchinvasion I haven't used this in a while but something makes me think there's a more declarative way of doing this using analytics.  Not a stopper on this pr - just worth looking at.

File: source/employers.html.haml [hunk 2]
```diff
@@ -162,3 +162,6 @@
         = image_tag logo, { alt: alt_text_from_filename(logo), title: alt_text_from_filename(logo) }
 
 = partial :"apply", locals: { page: :employers }
+:javascript
+  analytics.track('Viewed Employers Page');
```
Comment (silvabox) [inline source/employers.html.haml]: I think you can get Segment to track page views automatically - it might even do it by default.

**Comment by silvabox** [comment]
:shipit: unless you want to investigate comments above

File: source/employers.html.haml (hunk 2 above)
Comment (pitchinvasion) [inline source/employers.html.haml]: Yes you can but annoyingly it doesn't work with the way we've been doing analytics in the past, so I couldn't use it here

File: source/javascripts/link_tracking.js (hunk 1 above)
Comment (pitchinvasion) [inline source/javascripts/link_tracking.js]: I'd be interested to see that if you find a link at some point

File: source/javascripts/link_tracking.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  $('.employers-button').click(function(event) {
```
Comment (spike01) [inline source/javascripts/link_tracking.js]: document ready for safety?

File: source/javascripts/link_tracking.js (hunk 1 above)
Comment (silvabox) [inline source/javascripts/link_tracking.js]: https://segment.com/docs/libraries/analytics.js/#track-link

Comment (pitchinvasion) [inline source/javascripts/link_tracking.js]: awesome I'll change the code, much cleaner

**Pull request merged**